prospero-metadata/src/main/java/org/wildfly/prospero/metadata/ManifestVersionRecord.java

```diff
@@ -33,19 +34,35 @@
 import java.util.List;
 import java.util.Optional;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class ManifestVersionRecord {
 
+    protected static final String SCHEMA_100 = "1.0.0";
+    protected static final String DEFAULT_SCHEMA = SCHEMA_100;
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
     public static class MavenManifest {
 
+        private final String description;
         private String groupId;
         private String artifactId;
         private String version;
+
+        @Deprecated
+        public MavenManifest(String groupId,
+                             String artifactId,
+                             String version) {
+            this(groupId, artifactId, version, null);
+        }
+
         public MavenManifest(@JsonProperty("groupId") String groupId,
                              @JsonProperty("artifactId") String artifactId,
-                             @JsonProperty("version") String version) {
+                             @JsonProperty("version") String version,
+                             @JsonProperty(value = "description", required = false) String description) {
```

Comment:
I wanted to separate where the data is coming from in the manifest YAML from the field in record YAML, in case we decide to use a different manifest in the future. But yeah that's a bit confusing, I don't mind changing that to "name"